felix_requirements.txt

```diff
@@ -1,4 +1,4 @@
 gevent
 greenlet
 netaddr
-python-etcd==0.3.3-calico-1
+python-etcd==0.3.3-calico-2
```

Comment:
I had two thoughts in mind:
* putting the github link in here didn't seem to work when I tried it
* seemed wrong for our package itself to insist on getting the code from git rather than requiring a specific version
* I pegged it at that version since a 0.3.4 release from upstream would trump our fork's version.